backend/tournesol/views/suggestions/to_compare.py

```diff
@@ -38,16 +40,54 @@ class SuggestionsToCompare(PollScopedViewMixin, generics.ListAPIView):
 
     strategy = None
 
+    # When the user's preferred languages are unknown, use this language.
+    fallback_lang = "en"
+    # If the user's preferred languages don't contain at least one language
+    # from this list, use the fallback language.
+    min_lang_set = ["en", "fr"]
+
+    def _langs_from_accept_language(self, header: str) -> list[str]:
+        """
+        Return a list of language tags from a given Accept-Language HTTP
+        header.
+
+        See https://www.rfc-editor.org/rfc/rfc9110#field.accept-language
+        """
+        return [lang.split(';')[0].strip() for lang in header.split(",")]
```

Comment:
[`django.utils.translation.trans_real.parse_accept_lang_header`](https://github.com/django/django/blob/4.2.11/django/utils/translation/trans_real.py#L618) does something similar, possibly in a more robust way. But not sure this is a stable API.

Reply:
Addressed in https://github.com/tournesol-app/tournesol/pull/1961/commits/77102964668d47942c92cf8274e75d0820af1eb2